auth-proxy/conf/etc/httpd/conf.d/01-auth-proxy.conf

```diff
@@ -28,6 +28,10 @@ Listen 8080
  
     ProxyPass ${FILING_API_UPSTREAM_URI}
     ProxyPassReverse ${FILING_API_UPSTREAM_URI}
+
+    
+    # Prevent auth-proxy from setting bad x-forwarded-host
```

Comment:
I'm not sure it is accurate to say it was _`setting bad x-forwarded-host`_ header.  What we do know is that somewhere in the chain of reverse proxies, a space character is added into the comma-separated list of hostnames, and akka-http does not like that space.  I'm fairly certain this issue is not _solely_ on auth-proxy as we don't see the issue in all envs.  What we _do_ know is that not setting the `X-Forward-*` headers here does make the issue go away.  So, I would suggest adding a bit more detail as we may have to come back to this someday.  How about:

```apache
# We have encountered the following warning from our akka-http based APIs:
#     Illegal 'x-forwarded-host' header: Invalid input ' '
# This is caused by a space character being added to the comma-separated list
# of hostnames in the `X-Forwarded-Host` header by a reverse proxy somewhere
# in the request chain.  Setting `ProxyAddHeaders Off` is a brute-force means of
# removing the additional hostname.  This does make the warning go away, but it
# does mean our akka-http APIs (and any future apps put behind auth-proxy) will
# only see `X-Forwarded-*` headers set by the previous reverse proxy, not auth-proxy.
ProxyAddHeaders Off
```

Reply:
Sure, I'll throw that in.